app/src/main/java/com/iic/shopingo/ui/request_flow/Activities/CreateRequestActivity.java

```diff
@@ -0,0 +1,81 @@
+package com.iic.shopingo.ui.request_flow.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.ActionBarActivity;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import com.iic.shopingo.R;
+import com.iic.shopingo.ui.request_flow.views.CreateRequestItemListView;
+import com.iic.shopingo.ui.request_flow.views.CreateRequestListItemView;
+import java.util.Currency;
+import java.util.Locale;
+
+public class CreateRequestActivity extends ActionBarActivity implements
+    CreateRequestListItemView.OnListViewChanged {
+  public static final String REQUEST_EXTRA_KEY = "request";
```

Comment:
let's prefix those constants with EXTRA, EXTRA_REQUEST_KEY.
This way it's easy to find all of them in auto complete.